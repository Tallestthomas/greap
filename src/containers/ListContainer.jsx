import React from 'react';

const ListContainer = () => <div>Application</div>;

export default ListContainer;
