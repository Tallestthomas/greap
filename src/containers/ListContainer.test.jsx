import React from 'react';
import { render, cleanup } from 'react-testing-library';
import ListContainer from './ListContainer.jsx';

afterEach(cleanup);

describe('ListContainer Tests', () => {
  test('should render without crashing', () => {
    const { asFragment } = render(<ListContainer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
