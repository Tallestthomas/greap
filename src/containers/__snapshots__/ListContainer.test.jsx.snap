// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListContainer Tests should render without crashing 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
